src/commands/scheduled/recently-published.ts

```diff
@@ -173,5 +177,25 @@ function formatDate(dateString: string | undefined) {
     year: "numeric",
     month: "short",
     day: "numeric",
+    timeZone: "UTC",
   })
 }
+
+function formatWebURL(urlString: string | undefined) {
+  if (urlString === undefined) {
+    return ""
+  }
+
+  return urlString.replace(".mp3", "")
```

Comment:
buzzsprout's webpage for the episode is the same URL as the mp3 itself, without `.mp3`.